carma_wm_ros2/include/carma_wm_ros2/WorldModel.hpp

```diff
@@ -0,0 +1,418 @@
+#pragma once
```

Comment:
Please confirm that there are no changes to this interface other than the ros message namespace changes

Reply:
Yes, there are no other changes apart from that here